main.py

```diff
@@ -43,9 +43,9 @@ def main() -> None:
         pass
 
     if args.info is not None:
-        print_game_info(args.info)
+        print_game_info(args.info.lstrip('MatchID='))
     elif args.save is not None:
-        save_game_info(args.save[0], args.save[1])
+        save_game_info(args.save[0].lstrip('MatchID='), args.save[1])
```

Comment:
Also, maybe it makes more sense to pull it down from the presentation layer into the logic layer? So maybe make the call in `save_game_info`/`print_game_info`/`DataController.add`/`DataController.remove`. I'm not 100% sure, though, so maybe it's still better here. If you do move it, it should be easy to add tests for this case on the `DataController` class.